SPECS/libvirt/libvirt.spec

```diff
@@ -1,654 +1,1856 @@
+    %define with_qemu_kvm      %{with_qemu}
+%else
+    %define with_qemu_kvm      0
+%endif
+
+%define with_qemu_tcg      %{with_qemu}
+
+%if ! %{with_qemu_tcg} && ! %{with_qemu_kvm}
+    %define with_qemu 0
+%endif
+
+# Then the hypervisor drivers that run outside libvirtd, in libvirt.so
+%define with_openvz        0%{!?_without_openvz:1}
+%define with_vmware        0%{!?_without_vmware:1}
+%define with_esx           0%{!?_without_esx:1}
+%define with_hyperv        0%{!?_without_hyperv:1}
+
+# bfjelds: to maintain previous mariner settings, explicitly disable curl
+# and enable driver_esx by setting with_esx=0
+%define with_esx           0
+
+# Then the secondary host drivers, which run inside libvirtd
+%define with_storage_rbd      0%{!?_without_storage_rbd:1}
+
+%define with_storage_gluster 0%{!?_without_storage_gluster:1}
+
+%define with_storage_zfs      0
+
+%define with_storage_iscsi_direct 0%{!?_without_storage_iscsi_direct:1}
+# bfjelds: to maintain previous mariner settings, explicitly disable iscsi
+# by setting with_storage_iscsi_direct=0
+%define with_storage_iscsi_direct 0
+
+# Other optional features
+%define with_numactl          0%{!?_without_numactl:1}
+
+# A few optional bits off by default, we enable later
+%define with_fuse             0
+%define with_sanlock          0
+%define with_numad            0
+%define with_nbdkit           0
+%define with_nbdkit_config_default 0
+%define with_firewalld_zone   0
+%define with_netcf            0
+%define with_libssh2          0
+%define with_wireshark        0
+%define with_libssh           0
+%define with_dmidecode        0
+
+# Finally set the OS / architecture specific special cases
+
+# Architecture-dependent features
+%ifnarch %{arches_xen}
+    %define with_libxl 0
+%endif
+%ifnarch %{arches_vbox}
+    %define with_vbox 0
+%endif
+%ifnarch %{arches_numactl}
+    %define with_numactl 0
+%endif
+%ifnarch %{arches_zfs}
+    %define with_storage_zfs 0
+%endif
+%ifnarch %{arches_ceph}
+    %define with_storage_rbd 0
+%endif
+
+%define with_openvz 0
+%define with_vbox 0
+%define with_vmware 0
+%define with_libxl 0
+%define with_hyperv 0
+%define with_lxc 0
+
+%define with_netcf 0
+
+# fuse is used to provide virtualized /proc for LXC
+%if %{with_lxc}
+    %define with_fuse      0%{!?_without_fuse:1}
+%endif
+# bfjelds: to maintain previous mariner settings, explicitly enable fuse
+# by setting with_fuse=1
+%define with_fuse 1
+
+# Enable sanlock library for lock management with QEMU
+# Sanlock is available only on arches where kvm is available for RHEL
+%define with_sanlock 0%{!?_without_sanlock:1}
+
+# Enable libssh2 transport for new enough distros
+# bfjelds: to maintain previous mariner settings, explicitly enable libssh2
+# by setting with_libssh2=1
+%define with_libssh2 1
+
+# Enable wireshark plugins for all distros
+# bfjelds: to maintain previous mariner settings, explicitly disnable wireshark
+# by setting _with_wireshark=0
+%define with_wireshark 0
```

Comment:
mariner previously built without wireshark

Reply:
We now have azl and azl3 macros available for use https://github.com/microsoft/azurelinux/pull/8013. It will be good to set these build variables conditionally using the said macros.

Reply:
it seems like extra work to use the azl macro if the other distro macros have been removed.  i think for now, i'll leave the comments+hard-code.  long term, i think we should shift to leaving all macros in and adding azl special handling using these new azl macros.